app/views/chapters/show.html.erb

```diff
@@ -2,6 +2,78 @@
 
 <div class="main-content">
   <div class="pc-banner-full"></div>
+  <section class="story-see-footer has-background-white on" id= "nav-scroll">
+    <div class="container">
+      <div class="level">
+        <div class="level-left">
+          <ul class="list-01">
+            <li>
+              <%= link_to root_path do %>
+                <i class="fa fa-home"></i>
+                <span class="control-see"><%= t ".home" %></span>
+              <% end %>
+            </li>
+          </ul>
+        </div>
+        <div class="select-custom level">
+           <div class="prev level-right">
+            <% if @chapter.relate?(@chapter.prev) %>
+              <%= link_to comic_chapter_path(@chapter.comic, @chapter.prev) do %>
+                <i class="fa fa-arrow-circle-left"></i>
+              <% end %>
+            <% else %>
+              <%= link_to "#", class: "disable" do %>
+                <i class="fa fa-arrow-circle-left"></i>
+              <% end %>
+            <% end %>
+            </div>
+          <div class="dropdown select-custom">
+            <%= link_to "", class: "dropdown-toggle",
+                  "data-toggle": "dropdown"  do %>
+              <%= @chapter.name %>
+            <% end %>
+            <div class="dropdown-menu scrollable-menu">
+              <% @chapter.comic.chapters.each do |chapter| %>
```

Comment:
đoạn xử lý này hơi lòng vòng nhỉ, em tìm cách lấy luôn toàn bộ chapters có được k